src/scripts/itemdetailpage.js

```diff
@@ -1166,30 +1165,55 @@ define(["loading", "appRouter", "layoutManager", "connectionManager", "cardBuild
                     }))
                 }
             }
-            var currentItem, self = this,
-                apiClient = params.serverId ? connectionManager.getApiClient(params.serverId) : ApiClient;
+
+            var currentItem;
+            var self = this;
+            var apiClient = params.serverId ? connectionManager.getApiClient(params.serverId) : ApiClient;
```

Comment:
this changes the logic, before both `self` and `apiClient` were global variables, now they're local, this doesn't seem right.

Reply:
It's the same syntax in C. Multiple local variables are defined with the comma operator

Reply:
This is fine then right? I don't think the variables have changed scope.

Reply:
My bad :) TIL